src/main/java/io/jenkins/blueocean/maven/plugin/ProcessUpstreamDependenciesMojo.java

```diff
@@ -178,11 +178,14 @@ private void collectBlueoceanDependencies(@Nonnull DependencyNode node, @Nonnull
         boolean isLocalProject = node.getArtifact().equals(project.getArtifact());
         try {
             if (!isLocalProject) { // not the local project
-                if (getLog().isDebugEnabled()) getLog().debug("Testing artifact for Blue Ocean plugins: " + artifact.toString());
-                List<Contents> jarEntries = findJarEntries(artifact.getFile().toURI(), "package.json");
-                if (jarEntries.size() > 0) {
-                    getLog().info("Adding upstream Blue Ocean plugin: " + artifact.toString());
-                    results.add(artifact);
+                //filtering system they cannot contains package.json files
+                if(! artifact.getScope().equals("system")) {
```

Comment:
compare to [`Artifact.SCOPE_SYSTEM`](http://maven.apache.org/ref/3.8.1/maven-artifact/apidocs/org/apache/maven/artifact/Artifact.html#SCOPE_SYSTEM)?